typescript/sdk/src/providers/transactions/submitter/ethersV5/EV5TimelockSubmitter.ts

```
@@ -0,0 +1,166 @@
+        salt: config.salt ?? ZERO_32_BYTES,
+      },
+      multiProvider,
+      internalSubmitter,
+      timelockInstance,
+    );
+  }
+
+  async submit(
+    ...txs: AnnotatedEV5Transaction[]
+  ): Promise<
+    | void
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)